src/generators/jsx-ast/utils/__tests__/buildContent.test.mjs

```diff
@@ -0,0 +1,65 @@
+import assert from 'node:assert/strict';
+import { describe, it } from 'node:test';
+
+import { transformHeadingNode } from '../buildContent.mjs';
+
+describe('transformHeadingNode (deprecation Type -> AlertBox level)', () => {
+  const makeHeading = () => ({
+    type: 'heading',
+    depth: 3,
+    data: { type: 'misc', slug: 's', text: 'Heading' },
+    children: [{ type: 'text', value: 'Heading' }],
+  });
```

Comment:
Why does this half to be a function? Can't we just have a constant with the same purpose / inline?